### DISCUSSION TITLE
Logging failed login attempts?

### DISCUSSION THREAD

**Discussion by alexanderharm** [kimai/kimai#3954] [Questions & Answers] [answered]
We try improve the security of all our installations by banning IPs using fail2ban based on failed login attempts. However, in Kimai these login failures do not seem to be logged anywhere. How can I log them?

**Comment by kevinpapst** [comment, marked as answer]
There is a fantastic new technology existing. Yeah, not ChatGPT ... it is called "searching". 

Usually this a RTM, but in that case it is more of a LMGTFY: https://www.ecosia.org/search?q=kimai%20fail2ban

**Comment by alexanderharm** [comment]
Well, this is embarassing. I actually never search for kimai AND fail2ban just for kimai log failed login and such... thanks a lot and sorry for your time.

**Reply by kevinpapst** [reply]
Whatever you need, always check the store first. I'd say that more than 60% of all plugins are listed there: 
https://www.kimai.org/store/
And then follow the links to the GitHub categories, that will list some more.
